41_snapshots-btrfs

```diff
@@ -45,30 +45,34 @@ grub_btrfs_config="${sysconfdir}/default/grub-btrfs/config"
 
 ## Exit the script, if:
 disable_script() {
-    # Disable Grub-btrfs is set to true (default=false)
-    [[ "${GRUB_BTRFS_DISABLE:-"false"}" == "true" ]] && return 1
-    # btrfs-progs isn't installed
-    if ! type btrfs >/dev/null 2>&1; then return 1; fi
-    # grub-mkconfig_lib couldn't be found
-    [[ -f "$datarootdir/grub/grub-mkconfig_lib" ]] && . "$datarootdir/grub/grub-mkconfig_lib" || return 1
+    [[ "${GRUB_BTRFS_DISABLE:-"false"}" == "true" ]] && return 1 # Disable Grub-btrfs is set to true (default=false)
+    if ! type btrfs >/dev/null 2>&1; then return 1; fi # btrfs-progs isn't installed
+    [[ -f "$datarootdir/grub/grub-mkconfig_lib" ]] && . "$datarootdir/grub/grub-mkconfig_lib" || return 1 # grub-mkconfig_lib couldn't be found
     # Root filesystem isn't btrfs
     root_fs=$(${grub_probe} --target="fs" / 2>/dev/null)
     [[ "$root_fs" != "btrfs" ]] && return 1
     return 0
 }
 disable_script
 
+## Error Handling
+print_error()
+{
+    local err_msg="$*"
+    local bug_report="If you think an error has occurred , please file a bug report at \" https://github.com/Antynea/grub-btrfs \""
+    printf "%s\n" "${err_msg}" "${bug_report}" >&2 ;
+    exit 0
+}
+
+printf "Detecting snapshots ...\n" >&2 ;
+
 ## Submenu name
 distro=$(awk -F "=" '/^NAME=/ {gsub(/"/, "", $2); print $2}' /etc/os-release)
-submenuname=${GRUB_BTRFS_SUBMENUNAME:-"${distro:-Linux} snapshots"}
-## Prefix entry
-prefixentry=${GRUB_BTRFS_PREFIXENTRY:-"Snapshot:"}
+submenu_name=${GRUB_BTRFS_SUBMENUNAME:-"${distro:-Linux} snapshots"}
```

Comment:
This variable is referred to as `submenuname` (without the underscore) later on, in line 602.
Maybe also change the other occurence, or revert this one ?

Reply:
Good catch ! Thank you for your help !